docs/executor.rst

```diff
@@ -382,6 +382,15 @@ Resource requests and other job characteristics can be controlled via the follow
 * :ref:`process-queue`
 * :ref:`process-time`
 
+.. tip::
+  In Nextflow version 22.10 or later, it is possible to use ``clusterOptions`` with other resources directives such as
```

Comment:
Not understanding, why says `Nextflow version 22.10 or later`? this has been always possible. Missing something?

Reply:
I guess I should be more specific. With PBS Pro it's not possible to use `clusterOptions` to specify resource settings with `-l` if you also use the nextflow directives. They will conflict because PBS Pro expects only one resource line. Other executors work fine because they don't require a single resource line.

Reply:
Updated

Reply:
Ok, I understand it now